src/kiln_plugin.h

```diff
@@ -0,0 +1,255 @@
+/*******************************************************************************
+ *
+ * ██╗  ██╗██╗██╗     ███╗   ██╗
+ * ██║ ██╔╝██║██║     ████╗  ██║
+ * █████╔╝ ██║██║     ██╔██╗ ██║
+ * ██╔═██╗ ██║██║     ██║╚██╗██║
+ * ██║  ██╗██║███████╗██║ ╚████║
+ * ╚═╝  ╚═╝╚═╝╚══════╝╚═╝  ╚═══╝
+ *
+ * Kiln Ethereum Ledger App
+ * (c) 2022-2024 Kiln
+ *
+ * contact@kiln.fi
+ ********************************************************************************/
+
+#pragma once
+
+#include <string.h>
+
+#include "eth_plugin_interface.h"
+#include <ctype.h>
+
+#define PLUGIN_NAME "Kiln"
+
+// ADDRESS_STR_LEN is 0x + addr + \0
+#define ADDRESS_STR_LEN 43
+
+// ****************************************************************************
+// * SUPPORTED SELECTORS
+// ****************************************************************************
+
+// Available selectors:
+//
+// V1 selectors
+// --- 0. deposit()
+// --- 1. withdraw(validator_pubkey)
+// --- 2. withdrawEL(validator_pubkey)
+// --- 3. withdrawCL(validator_pubkey)
+// --- 4. batchWithdraw(validator_pubkeys)
+// --- 5. batchWithdrawEL(validator_pubkeys)
+// --- 6. batchWithdrawCL(validator_pubkeys)
+// --- 7. requestValidatorsExit(validator_pubkeys)
+//
+// V2 selectors
+// --- 8. stake()
+// --- 9. requestExit(shares_count)
+// --- 10. multiClaim(exit_queues, ticket_ids, cask_ids)
+// --- 11. claim(uint256[],uint32[],uint16)
```

Comment:
we don' use this function anywhere in the dApp, why is it here ? did we use it before ?

Reply:
it's meant to be used if you call the exit queue directly if we ever want to stop using the multiclaim function on integration contract